src/components/results/dynamicsimulation/dynamic-simulation-result-table.js

```diff
@@ -76,10 +95,35 @@ const DynamicSimulationResultTable = ({ nodeUuid, studyUuid }) => {
         );
     }
 
+    // messages to show when no data
+    const dynamicSimulationStatus = useSelector(
+        (state) => state.computingStatus[ComputingType.DYNAMIC_SIMULATION]
+    );
+    const messages = useIntlResultStatusMessages(intl, true);
+    const overlayMessage = useMemo(
+        () =>
+            getNoRowsMessage(
+                messages,
+                result,
+                dynamicSimulationStatus,
+                !isLoading
+            ),
+        [messages, result, dynamicSimulationStatus, isLoading]
+    );
+
     return (
         <>
-            <Box sx={styles.loader}>{isLoading && <LinearProgress />}</Box>
-            {result && (
+            {isLoading && (
+                <Box sx={styles.loader}>
+                    <LinearProgress />
+                </Box>
+            )}
+            {overlayMessage && (
+                <Box sx={styles.overlay}>
+                    <Typography variant={'body2'}>{overlayMessage}</Typography>
+                </Box>
+            )}
+            {!overlayMessage && (
```

Comment:
Use a ternary

Reply:
YES! it is better for an exclusive situation.. DONE!